README.md

```diff
@@ -41,6 +41,7 @@
 - Setting to provide custom set of catalog
 - Export single integration file into Maven project (Quarkus or Spring Boot)
 - Run single integration file locally
+- Deploy single integration file into Kubernetes cluster
```

Comment:
I think we could add Kubernetes/OpenShift to make it clearer that it works with OpenShift (which is even by default)